packages/core/src/core/client.test.ts

```diff
@@ -2405,6 +2405,75 @@ ${JSON.stringify(
       // Assert
       expect(mockCheckNextSpeaker).not.toHaveBeenCalled();
     });
+
+    it('should create linked abort signal and pass it to turn.run', async () => {
```

Comment:
nit: this is a low value test as it checks the setup not the behavior. I would remove to leave more room for useful tests.